packages/desktop-client/src/components/AppBackground.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { css } from 'glamor';
 
 import AnimatedLoading from '../icons/AnimatedLoading';
-import { colors } from '../style';
+import { colorsm } from '../style';
```

Comment:
(sorry, still not a full code review; just taking a quick peak at the changes)

Could yo do a global replace of `colorsm` -> `colors`? That should reduce the LOC changed by a good number. :)

Reply:
I'm cautious of doing this until I'm sure there won't be other UI changes to merge as it lets me easily identify components that use the "old" way of coloring.